tests/test_profile.py

```diff
@@ -178,6 +178,7 @@ async def test_summary(aresponses):
         assert summary["Ready to be Picked Up"] == 0
         assert summary["Returned"] == 0
         assert summary["Undelivered"] == 0
+        assert summary["Unknown"] == 0
```

Comment:
We should write a new test that explicitly tests a package with an unknown type.

Reply:
Done. Also the pre commit hook failed so I fixed some typings issues in the client.py file